### ISSUE TITLE
Launch movie n' menus work.  Then Starting New Game Starts in CV1 black.  Can see in game map only

### ISSUE THREAD

**Issue by redmantab** [Nibre/MotherVR#56]
I can see the 20th century fox movie, the amd etc in the rift CV1 headset.  The starting menu works fine.  When I click on play game, play alien isolation, play on easy difficulty, start new game, the game turns black and I can only press the button to bring up the map.  When I close the map, everything else is black.  I can hear stuff like the game is started but can only bring up the in game map/menus.  When I close them out the entire screen is black

Please help !! 

Thank you !

**Comment by RoundingError** [comment]
Had the same issue when launching through Steam or the Oculus hub. I reinstalled the game and mod and then launched the exe directly and it worked fine.

**Comment by redmantab** [comment]
Reinstalled the game thru steam? That's how I own it  


Sent via the Samsung Galaxy Note8, an AT&T 4G LTE smartphone
-------- Original message --------From: Hans <notifications@github.com> Date: 10/28/17  9:11 PM  (GMT-08:00) To: Nibre/MotherVR <MotherVR@noreply.github.com> Cc: redmantab <tbirkinbine@farmersagent.com>, Author <author@noreply.github.com> Subject: Re: [Nibre/MotherVR] Launch movie n' menus work. Then Starting New
  Game Starts in CV1 black. Can see in game map only (#56) 
Had the same issue when launching through Steam or the Oculus hub. I reinstalled the game and mod and then launched the exe directly and it worked fine.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Nibre/MotherVR","title":"Nibre/MotherVR","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Nibre/MotherVR"}},"updates":{"snippets":[{"icon":"PERSON","message":"@RoundingError in #56: Had the same issue when launching through Steam or the Oculus hub. I reinstalled the game and mod and then launched the exe directly and it worked fine."}],"action":{"name":"View Issue","url":"https://github.com/Nibre/MotherVR/issues/56#issuecomment-340236400"}}}
This e-mail message and any documents accompanying this transmission may contain confidential information and are intended solely for the addressee (s) named above. If you are not the intended addressee/recipient, any use of, disclosure, copying, distribution, or reliance on the contents of this e-mail information is strictly prohibited. Please reply to the sender advising of the error in transmission and immediately delete/destroy the message and any accompanying documents.

Farmers Group, Inc. and its subsidiaries and affiliates, including Farmers Financial Solutions, LLC, reserve the right to monitor and review the content of all e-mail communications and attachments sent or received by or from this address and to retain them in accordance with the applicable regulatory requirements. Securities are offered through Farmers Financial Solutions, LLC, 30801 Agoura Road, Building 1, Agoura Hills, California 91301. Member FINRA & SIPC.

**Comment by RoundingError** [comment]
Yeah, I reinstalled through steam just to make sure it wasn't messed up or anything, but then I launched the game directly through the .exe instead of through steam.

**Comment by redmantab** [comment]
Are you the creator dude? Thanks for all the help man. This type of mod is extremely cool and we are fortunate to have it! 


Sent via the Samsung Galaxy Note8, an AT&T 4G LTE smartphone
-------- Original message --------From: Hans <notifications@github.com> Date: 10/28/17  10:54 PM  (GMT-08:00) To: Nibre/MotherVR <MotherVR@noreply.github.com> Cc: redmantab <tbirkinbine@farmersagent.com>, Author <author@noreply.github.com> Subject: Re: [Nibre/MotherVR] Launch movie n' menus work. Then Starting New
  Game Starts in CV1 black. Can see in game map only (#56) 
Yeah, I reinstalled through steam just to make sure it wasn't messed up or anything, but then I launched the game directly through the .exe instead of through steam.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Nibre/MotherVR","title":"Nibre/MotherVR","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Nibre/MotherVR"}},"updates":{"snippets":[{"icon":"PERSON","message":"@RoundingError in #56: Yeah, I reinstalled through steam just to make sure it wasn't messed up or anything, but then I launched the game directly through the .exe instead of through steam. "}],"action":{"name":"View Issue","url":"https://github.com/Nibre/MotherVR/issues/56#issuecomment-340239592"}}}
This e-mail message and any documents accompanying this transmission may contain confidential information and are intended solely for the addressee (s) named above. If you are not the intended addressee/recipient, any use of, disclosure, copying, distribution, or reliance on the contents of this e-mail information is strictly prohibited. Please reply to the sender advising of the error in transmission and immediately delete/destroy the message and any accompanying documents.

Farmers Group, Inc. and its subsidiaries and affiliates, including Farmers Financial Solutions, LLC, reserve the right to monitor and review the content of all e-mail communications and attachments sent or received by or from this address and to retain them in accordance with the applicable regulatory requirements. Securities are offered through Farmers Financial Solutions, LLC, 30801 Agoura Road, Building 1, Agoura Hills, California 91301. Member FINRA & SIPC.

**Comment by AtomR** [comment]
I'm having the same issue. Did the game end up working in VR for you @redmantab?
I've tried launching using steam, from the explorer, with or without steamVR turned on. The result is the same, I can go past the menu, start a new game, but I see a few seconds of what looks to me like a cutscene and it cuts to black.
Currently downloading the game again see it reinstalling helps any.

**Comment by AtomR** [comment]
No luck so far.

**Comment by Nibre** [comment]
Try launching the game with the `-disableblinders` launch option, which disables the blacking effect that happens when you're head goes inside a wall/object. Does that change anything for you? Also, you can try it with the newest release and see if it's still happening

**Comment by naamapokemen** [comment]
I have the same problem. Everything is black and the game goes bordee windowed. I use SteamVR, but my headset is 3Glasses S1 2880x1440 resolution.

**Comment by NeonUniverse** [comment]
Have you tried what Nibre suggested? I had the same problem myself (with a Vive however) and it hasn't happened since. The type of headset shouldn't matter as long as the ratio for the screens are the same, that being said it could just be the game being weird for some reason and nothing to do with the mod.

**Comment by Nibre** [comment]
Can you try with the latest [Release](https://github.com/Nibre/MotherVR/releases)?

**Comment by redmantab** [comment]
Oh how fun

 

Yes.  I’ll give it a whirl in a day or two.  Did your new release fix issues like this?  Can you email me a link to latest release site?  TY!

**Comment by NeonUniverse** [comment]
It's here on Git, check under releases
